tests/integration/test_dictionaries_with_bad_structure/test.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from helpers.client import QueryRuntimeException
+from helpers.cluster import ClickHouseCluster
+
+DICTIONARY_FILES = [
+    "configs/dictionaries/invalid_dict.xml",
+    "configs/dictionaries/valid_dict.xml",
+]
+
+cluster = ClickHouseCluster(__file__)
+instance = cluster.add_instance("instance", dictionaries=DICTIONARY_FILES)
+
+VALID_DICT_NAME = "valid_dict"
+BAD_DICT_NAME = "invalid_dict"
+UNKNOWN_DATA_TYPE_EXCEPTION_STR = "DB::Exception: Unknown data type"
+
+
+@pytest.fixture(scope="module")
+def started_cluster():
+    try:
+        cluster.start()
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+
+def test_select_from_system_dictionaries_with_bad_dictionary(started_cluster):
```

Comment:
Consider to rename `test_select_from_system_dictionaries_with_bad_dictionary` with `test_select_from_system_dictionaries_with_invalid_dictionary`.